remote-release-notes/v1_97.md

```diff
@@ -18,4 +18,8 @@ We are continuing to improve this experience and would love to [hear your feedba
 
 A new setting has been introduced to install extensions into a remote if that extension is already present locally. This mirrors the setting already available in the Dev Containers extension.
 
-Configure with `setting(remote.SSH.defaultExtensionsIfInstalledLocally)`.
\ No newline at end of file
+Configure with `setting(remote.SSH.defaultExtensionsIfInstalledLocally)`.
+
+### Migration path for legacy server users
+
+As we are fast approaching the EOL for [legacy server](https://aka.ms/vscode-remote/faq/old-linux), February 2025 release v1.98 will be the last release supporting these servers. We have explored an option for the SSH extension to let users provide a custom build of glibc and libstdc++ required for connecting to our server. More details on this is documented in this [draft](https://github.com/microsoft/vscode-docs/pull/7953/files) with test builds available [here](https://github.com/microsoft/vscode/pull/235232#issuecomment-2615764717). We are hoping this will provide additional time to complete their migration to our list of supported distros without having to disable VSCode updates.
```

Comment:
"We are hoping this will provide additional time to complete their migration to our list of supported distros without having to disable VSCode updates."
Let's try to make simpler sentence structure, and have a simple call for action. Maybe something like:
"We hope that this workaround provides additional time to migrate to supported Linux distros without having to disable VS Code updates. If you have any feedback on the workaround flow, do comment [here](https://github.com/microsoft/vscode/issues/231623)."